README.md

```diff
@@ -16,7 +16,8 @@ examples.
 
   virtualenv venv
   source venv/bin/activate
-  pip install .
+  pip install -e .
+  pip install nose pylint pynt sphinx sphinx_rtd_theme testfixtures
```

Comment:
Maybe we could move what's needed for testing and what's needed for docs into the two sections below? None of this is required to simply use it.